README.md

```diff
@@ -10,7 +10,7 @@ Golint requires Go 1.6 or later.
 
 ## Usage
 
-Invoke `golint` with one or more filenames, a directory, or a package named
+Invoke `golint` with one or more filenames, directories, or packages named
```

Comment:
I applied the change by hand to another repo and then extracted it back out to here. This was a transcribing error on my part. Thanks for pointing it out, I'll fix it on Monday. I hope to make this repo be able to accept PRs directly in the near future.